### ISSUE TITLE
[Feature Request] Set `lisp` option

### ISSUE THREAD

**Issue by alexmozaidze** [clojure-vim/clojure.vim#40]
Some plugins use `set lisp?` to determine if the file is Lisp or not. It would be cool if you had `lisp` option set just so that outside plugins may detect Clojure as Lisp. It's possible for outside plugins to hardcode Clojure as Lisp, but I don't think that's the best approach.

**Comment by axvr** [comment]
Thanks for the suggestion.  This will be included once #31 is merged into the `master` branch.
